processor/add_cluster_id/add_cluster_id.go

```diff
@@ -40,41 +41,51 @@ func init() {
 }
 
 type processor struct {
-	config config
```

Comment:
Let's discuss it, this file should be reverted.
Processors are dynamically added by the configuration.

Reply:
Agree, removed from local config.